src/nodes.coffee

```diff
@@ -3097,6 +3114,13 @@ exports.If = class If extends Base
 
 UTILITIES =
   modulo: -> 'function(a, b) { return (+a % (b = +b) + b) % b; }'
+  _boundMethodCheck: -> "
```

Comment:
helper that throws runtime error if bound method called with a `this` that isn't an instance of its own class

as discussed in coffeescript6/discuss#84, this results in a stack trace where the offending method is in the second line